File: src/com/app/test/annotation/OnceClickConfig.java
package com.app.test.annotation;

public class OnceClickConfig {
    private static final long DEFAULT_DURING = 1000L;
    private static long during;

    public OnceClickConfig() {
    }

    public static Long getDuring() {
        if (during <= 0L) {
            during = 1000L;
        }

        return during;
    }

    public static void init(boolean debug) {
        init(1000L, debug);
    }

    public static void init(long during, boolean debug) {
        OnceClickConfig.during = during;
//        OnceClickLogger.debug(debug);
    }
}

File: app/src/main/java/com/app/test/annotation/OnceClickConfig.kt
package com.app.test.annotation

object OnceClickConfig {
    private const val DEFAULT_DURING = 1000L
    private var during: Long = 0
    @JvmStatic
    fun getDuring(): Long {
        if (during <= 0L) {
            during = 1000L
        }
        return during
    }

    fun init(debug: Boolean) {
        init(1000L, debug)
    }

    fun init(during: Long, debug: Boolean) {
        OnceClickConfig.during = during
        //        OnceClickLogger.debug(debug);
    }
}